### PR TITLE
refactor: split clients into integrations and protocols

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

1. Split clients into integrations and protocols.
image

2. `/user-guide/clients/authentication.md` has been moved to `/operations/authentication.md`, we can adjust its location in the `operations` refactor PR in the future.

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/db-cloud-shared/migrate/migrate-from-influxdb.md [hunk 1]
```diff
@@ -2,61 +2,7 @@
...
-| Column    | Type                 | Key  | Null | Default | Semantic Type |
-+-----------+----------------------+------+------+---------+---------------+
-| location  | String               | PRI  | YES  |         | TAG           |
-| scientist | String               | PRI  | YES  |         | TAG           |
-| bees      | Float64              |      | YES  |         | FIELD         |
-| ts        | TimestampNanosecond  | PRI  | NO   |         | TIMESTAMP     |
-| ants      | Float64              |      | YES  |         | FIELD         |
-+-----------+----------------------+------+------+---------+---------------+
-```
+要了解 InfluxDB 和 GreptimeDB 的数据模型之间的差异，请参考从写入数据文档中的[数据模型](/user-guide/ingest-data/for-iot/influxdb-line-protocol.md#数据模型)。
```
Comment (killme2008) [inline i18n/zh/docusaurus-plugin-content-docs/current/db-cloud-shared/migrate/migrate-from-influxdb.md:5]:
```suggestion
要了解 InfluxDB 和 GreptimeDB 的数据模型之间的差异，请参考写入数据文档中的[数据模型](/user-guide/ingest-data/for-iot/influxdb-line-protocol.md#数据模型)。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/protocols/postgresql.md [hunk 2]
```diff
@@ -2,28 +2,23 @@
...
 ```
 
-记得将示例中的 `greptime_user(username)` 和 `greptime_pwd(password)` 替换成自己的用户名和密码。
+- 请参考[鉴权认证](/user-guide/operations/authentication.md) 来设置 GreptimeDB 的用户名和密码。
+- 如果你想使用其他端口连接 PostgreSQL，请参考配置文档中的[协议选项](/user-guide/operations/configuration.md#协议选项)。
 
 ## 管理表
 
-请参考 [管理表](../table-management.md)。
+请参考 [管理表](/user-guide/table-management.md)。
```
Comment (killme2008) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/protocols/postgresql.md:17]:
```suggestion
请参考[管理表](/user-guide/table-management.md)。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/protocols/postgresql.md [hunk 3]
```diff
@@ -2,28 +2,23 @@
...
 
 ## 写入数据
 
-请参考 [写入数据](../ingest-data/for-iot/sql.md).
+请参考 [写入数据](/user-guide/ingest-data/for-iot/sql.md).
```
Comment (killme2008) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/protocols/postgresql.md:21]:
```suggestion
请参考[写入数据](/user-guide/ingest-data/for-iot/sql.md).
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/upgrade.md [hunk 4]
```diff
@@ -136,7 +136,7 @@ greptime cli export --addr '127.0.0.1:4000' --output-dir /tmp/greptimedb-export
 
 ### 导入表结构和数据
 
-然后您需要执行上一步生成的 SQL 文件。首先是 `create_tables.sql`。在之前的步骤中导出的 SQL 语句使用的是 PostgreSQL 方言，接下来的操作都将通过 [PG 协议](/user-guide/clients/postgresql.md)来进行。本文档假设客户端为 `psql`。
+然后您需要执行上一步生成的 SQL 文件。首先是 `create_tables.sql`。在之前的步骤中导出的 SQL 语句使用的是 PostgreSQL 方言，接下来的操作都将通过 [PG 协议](/user-guide/protocols/postgresql.md)来进行。本文档假设客户端为 `psql`。
```
Comment (killme2008) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/upgrade.md:139]:
```suggestion
然后您需要执行上一步生成的 SQL 文件。首先是 `create_tables.sql`。在之前的步骤中导出的 SQL 语句使用的是 PostgreSQL 方言，接下来的操作都将通过 [PostgreSQL 协议](/user-guide/protocols/postgresql.md)来进行。本文档假设客户端为 `psql`。
```

File: docs/user-guide/upgrade.md [hunk 5]
```diff
@@ -135,7 +135,7 @@ The content includes `create_tables.sql`, `copy_from.sql`, and the parquet files
 
 ### Import table schema and data
 
-Then you need to execute SQL files generated by the previous step. First is `create_tables.sql`. SQL generated in previous step is in PostgreSQL dialect, we will use [PG protocol](/user-guide/clients/postgresql.md) in the following steps. This document assumes the client is `psql`.
+Then you need to execute SQL files generated by the previous step. First is `create_tables.sql`. SQL generated in previous step is in PostgreSQL dialect, we will use [PG protocol](/user-guide/protocols/postgresql.md) in the following steps. This document assumes the client is `psql`.
```
Comment (killme2008) [inline docs/user-guide/upgrade.md:138]:
```suggestion
Then you need to execute SQL files generated by the previous step. First is `create_tables.sql`. SQL generated in previous step is in PostgreSQL dialect, we will use [PostgreSQL protocol](/user-guide/protocols/postgresql.md) in the following steps. This document assumes the client is `psql`.
```

File: docs/user-guide/ingest-data/for-iot/influxdb-line-protocol.md [hunk 6]
```diff
@@ -68,9 +137,64 @@ To configure Telegraf, simply add GreptimeDB URL into Telegraf configurations:
...
+While you may already be familiar with [InfluxDB key concepts](https://docs.influxdata.com/influxdb/v2/reference/key-concepts/), the [data model](/user-guide/concepts/data-model.md) of GreptimeDB is something new to explore.
+Here are the similarities and differences between the data models of GreptimeDB and InfluxDB:
+
+- Both solutions are [schemaless](/user-guide/ingest-data/overview.md#automatic-schema-generation), eliminating the need to define a schema before writing data.
+- In InfluxDB, a point represents a single data record with a measurement, tag set, field set, and a timestamp.
+In GreptimeDB, it is represented as a row of data in the time-series table,
+where the table name aligns with the measurement,
+and the columns are divided into three types: Tag, Field, and Timestamp.
+- GreptimeDB uses `TimestampNanosecond` as the data type for timestamp data from the [InfluxDB line protocol API](/user-guide/ingest-data/for-iot/influxdb-line-protocol.md).
+- GreptimeDB uses `Float64` as the data type for numeric data from the InfluxDB line protocol API.
```
Comment (killme2008) [inline docs/user-guide/ingest-data/for-iot/influxdb-line-protocol.md:149]: Another key point is that greptimedb tables for InfluxDB will set the `'merge_mode'='last_non_null'` table option to keep the behavior the same as InfluxDB. It's worth to be noted here.

**Review by killme2008 — COMMENTED** [review]
Great work! Thanks a lot.

**Review by sunng87 — APPROVED** [review]

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**